library/bignum_core.h

```diff
@@ -262,6 +262,21 @@ int mbedtls_mpi_core_write_be( const mbedtls_mpi_uint *A,
                                unsigned char *output,
                                size_t output_length );
 
+/** \brief              Shift a machine integer right by a number of bits.
```

Comment:
Minor: I know what you mean by "machine integer", but we still call them MPIs in the new modules (it might sound ambiguous, but the parameter types make  it clear, and I admit that it is not ideal like this).

Reply:
This is actually misleading: a machine integer would be a single limb. So I'll fix it.